packages/fxa-settings/src/components/ButtonDownloadRecoveryKeyPDF/index.tsx

```diff
@@ -148,7 +148,7 @@ export const ButtonDownloadRecoveryKeyPDF = ({
     <>
       <FtlMsg id="recovery-key-download-button-v3">
         <CmsButtonWithFallback
-          className="cta-primary cta-xl w-full mt-4"
+          className="cta-xl w-full mt-4"
```

Comment:
Looks like `cta-xl` is also redundant here (already applied by default in CmsButtonWithFallback) based on the snap